### PR TITLE
Add config for sem_sachplan_asyl

### PR DESCRIPTION
**Pull request by Sgachet**

### CODE REVIEW

File: conf/dritte.conf.part [hunk 1]
```diff
@@ -36,6 +36,43 @@ source src_ch_mobility_standorte : def_searchable_features
...
+            , stabil_id as label \
+            , 'feature' as origin \
+            , remove_accents(coalesce(facname_de, ' ')||' '||coalesce(facname_fr, ' '))||' '||coalesce(facname_it, ' ') as  detail \
+            , 'ch.sem.sachplan-asyl_anhoerung' as layer \
+            , quadindex(the_geom) as geom_quadindex \
+            , st_y(st_transform(st_centroid(the_geom),4326)) as lat \
+            , st_x(st_transform(st_centroid(the_geom),4326)) as lon \
+            , box2d(st_transform(the_geom, 21781)) as geom_st_box2d \
+            , box2d(st_transform(the_geom, 2056)) as geom_st_box2d_lv95 \
+            , bgdi_id::text as feature_id \
```
Comment (loicgasser) [inline conf/dritte.conf.part]: why don't you use stabil_id here instead?

File: conf/dritte.conf.part [hunk 2]
```diff
@@ -36,6 +36,43 @@ source src_ch_mobility_standorte : def_searchable_features
...
+}
+
+source src_ch_sem_sachplan_asyl_facilities_kraft : def_searchable_features
+{
+    sql_db = dritte_dev
+    sql_query = \
+        SELECT bgdi_id as id \
+            , stabil_id as label \
+            , 'feature' as origin \
+            , remove_accents(coalesce(facname_de, ' ')||' '||coalesce(facname_fr, ' '))||' '||coalesce(facname_it, ' ') as  detail \
```
Comment (loicgasser) [inline conf/dritte.conf.part]: you can use `remove_accent(concat_ws(' ', facname_de, facname_fr, facname_it))` instead

File: conf/dritte.conf.part [hunk 3]
```diff
@@ -36,6 +36,43 @@ source src_ch_mobility_standorte : def_searchable_features
...
 }
 
+source src_ch_sem_sachplan_asyl_facilities_anhoerung : def_searchable_features
+{
+    sql_db = dritte_dev
+    sql_query = \
+        SELECT bgdi_id as id \
+            , stabil_id as label \
+            , 'feature' as origin \
+            , remove_accents(coalesce(facname_de, ' ')||' '||coalesce(facname_fr, ' '))||' '||coalesce(facname_it, ' ') as  detail \
```
Comment (loicgasser) [inline conf/dritte.conf.part]: you can use `remove_accent(concat_ws(' ', facname_de, facname_fr, facname_it))` instead

File: conf/dritte.conf.part [hunk 4]
```diff
@@ -36,6 +36,43 @@ source src_ch_mobility_standorte : def_searchable_features
...
+            , 'ch.sem.sachplan-asyl_kraft' as layer \
+            , quadindex(the_geom) as geom_quadindex \
+            , st_y(st_transform(st_centroid(the_geom),4326)) as lat \
+            , st_x(st_transform(st_centroid(the_geom),4326)) as lon \
+            , box2d(st_transform(the_geom, 21781)) as geom_st_box2d \
+            , box2d(st_transform(the_geom, 2056)) as geom_st_box2d_lv95 \
+            , bgdi_id::text as feature_id \
```
Comment (loicgasser) [inline conf/dritte.conf.part]: stabil_id

File: conf/dritte.conf.part (hunk 1 above)
Comment (Sgachet) [reply conf/dritte.conf.part]: Ooops, i forgot to change

**Comment by Sgachet** [comment]
@loicgasser Thanks for the review. Changes are done

**Comment by loicgasser** [comment]
thank you for your fast fix.

**Pull request merged**